Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionContextCocoa.mm

```diff
@@ -527,6 +531,9 @@ static _WKWebExtensionContextError toAPI(WebExtensionContext::Error error)
     if (permissions.isEmpty())
         return;
 
+    if (m_nextGrantedPermissionsExpirationDate > expirationDate)
```

Comment:
Super nit: I think this is a bit more readable as `expirationDate < m_nextGrantedPermissionsExpirationDate`.